ZukoAzula2016/src/org/team1540/zukoazula/AutonomousBase.java

```diff
@@ -97,6 +98,17 @@ protected void waitUntilArmStopped() throws AutonomousModeOverException, Interru
         waitUntil(IntakeArm.armIsStopped());
     }
 
+    protected void movePortcullisArmToPosition(float position, float speed) throws AutonomousModeOverException, InterruptedException {
+        if (position > Portcullis.getArmHeight().get()) {
+            Portcullis.getPortcullisOutput().set(Math.abs(speed));
+            waitUntilAtLeast(Portcullis.getArmHeight(), position);
+        } else {
+            Portcullis.getPortcullisOutput().set(-Math.abs(speed));
+            waitUntilAtMost(Portcullis.getArmHeight(), position);
```

Comment:
Maybe you want to leave some wiggle room in case the arm can't quite get to the target position?